src/frontend/Sider/Journalføring/Standard/Journalføring.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useEffect } from 'react';
+
+import { Navigate } from 'react-router-dom';
+import styled from 'styled-components';
+
+import { Heading } from '@navikt/ds-react';
+
+import Dokumenter from './Dokumenter';
+import { useQueryParams } from '../../../hooks/felles/useQueryParams';
+import { useHentJournalpost } from '../../../hooks/useHentJournalpost';
+import { JournalføringState, useJournalføringState } from '../../../hooks/useJournalføringState';
+import DataViewer from '../../../komponenter/DataViewer';
+import { JournalpostResponse } from '../../../typer/journalpost';
+import {
+    GJELDER_KLAGE_QUERY_STRING,
+    JOURNALPOST_QUERY_STRING,
+    OPPGAVEID_QUERY_STRING,
+} from '../../Oppgavebenk/oppgaveutils';
+import PdfVisning from '../Felles/PdfVisning';
+
+export const Kolonner = styled.div`
+    display: flex;
+    flex-wrap: wrap;
+    align-items: stretch;
+`;
+
+export const Venstrekolonne = styled.div`
+    padding: 1rem 2rem 1rem 2rem;
+    height: inherit;
+    overflow: auto;
+    @media (min-width: 1225px) {
+        height: calc(100vh - 4rem);
+        max-width: 750px;
+    }
+`;
+export const Høyrekolonne = styled.div`
+    display: flex;
+    flex: 1 1 auto;
+    min-width: 450px;
+    height: calc(100vh - 4rem);
+`;
+
+const InnerContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+    gap: 1.25rem;
+    min-width: 42.75rem;
+`;
+
+const Tittel = styled(Heading)`
+    margin-bottom: 0.5rem;
+`;
+
+export const Journalføring: React.FC = () => {
+    const query: URLSearchParams = useQueryParams();
+    const oppgaveId = query.get(OPPGAVEID_QUERY_STRING);
+    const journalpostId = query.get(JOURNALPOST_QUERY_STRING);
+    const gjelderKlage = query.get(GJELDER_KLAGE_QUERY_STRING) === 'true';
+
+    const { hentJournalPost, journalResponse } = useHentJournalpost();
+
+    useEffect(() => {
+        document.title = 'Journalpost';
+        hentJournalPost(journalpostId);
+    }, [hentJournalPost, journalpostId]);
+
+    // TODO: Kan vurdere å vise en infostripe med feilmelding her
+    if (!oppgaveId || !journalpostId) {
+        return <Navigate to="/oppgavebenk" />;
+    }
+
+    return (
+        <DataViewer response={{ journalResponse }}>
+            {({ journalResponse }) => {
+                return (
+                    <JournalføringSide
+                        oppgaveId={oppgaveId}
+                        gjelderKlage={gjelderKlage}
+                        journalResponse={journalResponse}
+                    />
+                );
+            }}
+        </DataViewer>
+    );
+};
+
+interface Props {
+    oppgaveId: string;
+    gjelderKlage: boolean;
+    journalResponse: JournalpostResponse;
+}
+
+const JournalføringSide: React.FC<Props> = ({ journalResponse }) => {
+    const journalpostState: JournalføringState = useJournalføringState(journalResponse);
+
+    return (
+        <Kolonner>
+            <Venstrekolonne>
```

Comment:
Kan bruke grid her for å slippe å neste mange flexbokser. Noe ala

```javascript
const Container = styled.div`
    display: grid;
    grid-template-columns: repeat(auto-fit, minmax(*X*px, 1fr));
    gap: 1rem;
    height: calc(100vh - 4rem);
`;
``
`

Reply:
Altså bytte ut `Kolonner` med denne? Vet ikke helt om det gir samme visning siden vi bruker en iFrame til å vise frem pdfen her